signac/__main__.py

```diff
@@ -19,6 +19,8 @@
 import errno
 from pprint import pprint, pformat
 
+from packaging import version
```

Comment:
https://stackoverflow.com/a/11887885
According to this StackOverflow post, `packaging` isn't in the standard library but comes as a dependency of `setuptools`. I just wanted to make a note of that.

Reply:
Yes, so we should probably add `setuptools` to the dependencies, however, it is not clear to me if we need to require a specific version.

Reply:
@csadorf I looked at the history of the setuptools git repo and changelog. `packaging` has been included as a vendored package (not a dependency) since setuptools v8.0 (Dec 13, 2014). I think that StackOverflow answer I linked above is out of date. I recommend that we add `packaging` as a dependency instead. It's a very common dependency so I expect it will be already met by most users' environments.

Reply:
But then in [v39](https://setuptools.readthedocs.io/en/latest/history.html#v39-0-0) it was changed to dependencies again?

Just tested, `packaging` is not installed by default in a "fresh" conda environment, but it's certainly very easy to install.

Reply:
I tested this and we need `packaging>=15.0`.